homework27/src/ba/bitcamp/homework/tasks/Task2.java

```diff
@@ -0,0 +1,164 @@
+package ba.bitcamp.homework.tasks;
+
+import java.awt.Graphics;
+import java.awt.image.BufferedImage;
+import java.io.File;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.concurrent.LinkedBlockingQueue;
+
+import javax.imageio.ImageIO;
+import javax.swing.JFrame;
+import javax.swing.JPanel;
+
+
+/**
+ * This class extends JFrame and it is used for making negative from readed
+ * picture. Picture is being shown using GUI
+ * 
+ *
+ */
+public class Task2 extends JFrame {
+	// Declaring variables
+	private static final long serialVersionUID = -5118980704703412553L;
+	private static BufferedImage img = null;
+	private JPanel panel;
+	private static LinkedBlockingQueue<Runnable> queue;
```

Comment:
Using static modifier here is not advised. Static means that in case there are more than two instances of the same class, state is shared which means it can lead to many problems down the line.